examples/pln/johnny/Assertions.scm

```diff
@@ -0,0 +1,50 @@
+(ConceptNode "Johnny@444") (stv 0.99 1)
+
+(ConceptNode "Sweets@555") (stv 0.99 1)
+
+(ConceptNode "day@222") (stv 0.99 1)
+
+(ConceptNode "he@345") (stv 0.99 1)
+
+(ConceptNode "himself@456") (stv 0.99 1)
+
+(ConceptNode "risk@44") (stv 0.99 1)
+
+(ConceptNode "diabetes@66") (stv 0.99 1)
+
+(ImplicationLink (stv 0.99 1)
+	(EvaluationLink
+		(PredicateNode "every@111")
+        (ListLink
+			(ConceptNode "day@222")
+			(EvaluationLink
+				(PredicateNode "eating@333")
+				(ListLink
+					(ConceptNode "Johnny@444")
+					(ConceptNode "Sweets@555")))))
+	(EvaluationLink
+		(PredicateNode "at@123")
+		(ListLink
+			(EvaluationLink
+				(PredicateNode "placing@234")
+				(ListLink
+					(ConceptNode "he@345")
+					(ConceptNode "himself@456")))
+			(ConceptNode "risk@44"))))
+
+(EvaluationLink (stv 0.99 1)
+	(PredicateNode "for@55")
+	(ListLink
+		(ConceptNode "risk@44")
+		(ConceptNode "diabetes@66")))
+
+(NotLink (stv 0.99 1)
+	(EvaluationLink
+		(PredicateNode "every@111")
+        (ListLink
+			(ConceptNode "day@222")
+			(EvaluationLink
+				(PredicateNode "eating@333")
+				(ListLink
+					(ConceptNode "Johnny@444")
+					(ConceptNode "Sweets@555"))))))
```

Comment:
Okay, that's done now . . . just waiting to be merged, which may take a
while, because there were some commits stacked up there.

On Tue, Jun 16, 2015 at 11:14 AM, Aaron Nitzkin <a_nitzkin@yahoo.com> wrote:

> Nope. I keep getting the 403 error and I have not been able to correct it
> yet. Will keep trying.
>
> On Tue, Jun 16, 2015 at 5:05 AM, Prateek Saxena <notifications@github.com>
> wrote:
>
>> In examples/pln/johnny/Assertions.scm
>> <https://github.com/opencog/opencog/pull/1594#discussion_r32466716>:
>>
>> > +(EvaluationLink (stv 0.99 1)
>> > +	(PredicateNode "for@55")
>> > +	(ListLink
>> > +		(ConceptNode "risk@44")
>> > +		(ConceptNode "diabetes@66")))
>> > +
>> > +(NotLink (stv 0.99 1)
>> > +	(EvaluationLink
>> > +		(PredicateNode "every@111")
>> > +        (ListLink
>> > +			(ConceptNode "day@222")
>> > +			(EvaluationLink
>> > +				(PredicateNode "eating@333")
>> > +				(ListLink
>> > +					(ConceptNode "Johnny@444")
>> > +					(ConceptNode "Sweets@555"))))))
>>
>> Hello @anitzkin <https://github.com/anitzkin>, have you been able to
>> push your changes?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/opencog/opencog/pull/1594/files#r32466716>.
>>
>
>